### ISSUE TITLE
SERVER_PORT does not follow WSGI specs

### ISSUE THREAD

**Issue by Varbin** [jonashaag/bjoern#122]
The WSGI specification says, it must be a string (https://www.python.org/dev/peps/pep-3333/#environ-variables). This prevents running application wrapped with `wsgiref.validate.validator`.

Traceback:

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/wsgiref/validate.py", line 153, in lint_app
    check_environ(environ)
  File "/usr/lib/python3.6/wsgiref/validate.py", line 328, in check_environ
    % (key, type(environ[key]), environ[key]))
  File "/usr/lib/python3.6/wsgiref/validate.py", line 128, in assert_
    raise AssertionError(*args)
AssertionError: Environmental variable SERVER_PORT is not a string: <class 'int'> (value: 8000)
```

Shall I fix it and write a pull request?

**Comment by jonashaag** [comment]
Yes please!

**Comment by Varbin** [comment]
Would you mind if I add a test if bjoern does follow WSGI specification?

**Comment by jonashaag** [comment]
Yes please! I even started a WSGI test suite at some point (that's more extensive than the wsgiref one) but abandoned the project for some reason I can't recall right now. So um all for WSGI compliance tests :-)

**Comment by Varbin** [comment]
The simplest implementation would be to just run a wsgiref-validated (obviously not complete but at least a reference) app in bjoern. This is now implemented with 14de36ff88b35968679e34b2b25d27dc8589fe07. While it shows a warning it seems working.

**Comment by Varbin** [comment]
I guess this can be safely closed now.
